upup/pkg/fi/cloudup/awstasks/load_balancer.go

```diff
@@ -323,3 +331,79 @@ func (_ *LoadBalancer) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *LoadBalan
 
 	return t.AddELBTags(*e.ID, t.Cloud.BuildTags(e.Name))
 }
+
+type terraformLoadBalancer struct {
+	Name           *string                           `json:"name"`
+	Listener       []*terraformLoadBalancerListener  `json:"listener"`
+	SecurityGroups []*terraform.Literal              `json:"security_groups"`
+	Subnets        []*terraform.Literal              `json:"subnets"`
+	Internal       bool                              `json:"internal,omitempty"`
+	HealthCheck    *terraformLoadBalancerHealthCheck `json:"health_check"`
+}
+
+type terraformLoadBalancerListener struct {
+	InstancePort     int    `json:"instance_port"`
+	InstanceProtocol string `json:"instance_protocol"`
+	LBPort           int64  `json:"lb_port"`
+	LBProtocol       string `json:"lb_protocol"`
+}
+
+type terraformLoadBalancerHealthCheck struct {
+	Target             *string `json:"target"`
+	HealthyThreshold   *int64  `json:"healthy_threshold"`
+	UnhealthyThreshold *int64  `json:"unhealthy_threshold"`
+	Interval           *int64  `json:"interval"`
+	Timeout            *int64  `json:"timeout"`
+}
+
+func (_ *LoadBalancer) RenderTerraform(t *terraform.TerraformTarget, a, e, changes *LoadBalancer) error {
+	elbName := e.ID
+	if elbName == nil {
+		elbName = e.Name
+	}
+
+	tf := &terraformLoadBalancer{
+		Name:     elbName,
+		Internal: *e.Scheme == "internal",
```

Comment:
I suspect `fi.StringValue(e.Schema) == "internal"` would be safer, in case Scheme is not set.  I presume the default value is a public ELB.